.nycrc

```diff
@@ -1,7 +1,14 @@
 {
     "include": [
-      "packages/*/dist*/*",
-      "examples/*/dist*/*"
+      "packages/*/dist*/",
+      "examples/*/dist*/"
+    ],
+    "exclude": [
+      "packages/*/dist*/test/",
+      "packages/*/test/",
+      "examples/*/dist*/test/",
+      "examples/*/test/",
+      "**/.sandbox/"
```

Comment:
(Some) bootstrapper tests are creating the sandbox in `packages/test/.sandbox`, some tests in another location that's not covered by the two patterns above. Ignoring all `.sandbox` directories seems like the safest option to me.